mslib/mscolab/file_manager.py

```diff
@@ -399,7 +400,7 @@ def get_all_changes(self, op_id, user, named_version=False):
             'comment': change.comment,
             'version_name': change.version_name,
             'username': change.user.username,
-            'created_at': change.created_at.strftime("%Y-%m-%d, %H:%M:%S.%f %z")
+            'created_at': created_at_isoformat(change.created_at)
```

Comment:
What is wrong with `change.created_at.isoformat()`?

Reply:
seems I looked only on that with the None state.

Reply:
What None state?

Reply:
on a test ?